lib/migrate_assets_to_asset_manager.rb

```diff
@@ -40,7 +42,7 @@ def asset_exists?(file)
   end
 
   class AssetFilePaths
-    delegate :each, :join, to: :file_paths
+    delegate :each, :join, :size, to: :file_paths
```

Comment:
Minor: I think you could stop delegating `join` given that it's no longer used in `MigrateAssetsToAssetManager#to_s`.

Reply:
I'll remove that and rebase.